frontend/src/libs/components/comment-card/comment-card.tsx

```diff
@@ -0,0 +1,39 @@
+import { Avatar, Icon, Link } from '~/libs/components/components.js';
+import { AppRoute } from '~/libs/enums/enums.js';
+import { type Comment } from '~/libs/types/comment.type.js';
+import { type UserAuthResponseDto } from '~/packages/users/users.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  user: UserAuthResponseDto;
+  comment: Comment;
+};
+
+const CommentCard: React.FC<Properties> = ({ user, comment }) => {
+  const { firstName, lastName } = user;
+  const { text, createdAt } = comment;
+  const userName = `${firstName} ${lastName}`;
+
+  return (
+    <section className={styles.comment}>
+      <div className={styles.header}>
+        <div className={styles.userInfo}>
+          <Link to={AppRoute.PROFILE}>
+            <Avatar username={userName} avatarUrl={null} />
+          </Link>
```

Comment:
In comment it should not be a link - just avatar
```suggestion
            <Avatar username={userName} avatarUrl={null} />
```